### DISCUSSION TITLE
Brand new instance returns only Mastodon icon

### DISCUSSION THREAD

**Discussion by AnibalDuarte** [mastodon/mastodon#21449] [Q&A]
Good night. 

First of all, I'm running on Ubuntu 22.04.1 LTS at Oracle Cloud.

I've just did all steps on the docs:

- https://docs.joinmastodon.org/user/run-your-own/ 
- https://docs.joinmastodon.org/admin/prerequisites/ 
- https://docs.joinmastodon.org/admin/install/ 
- https://docs.joinmastodon.org/admin/config/ 

Now I'm able to access it but only the Mastodon icon are beign shown: https://mastodonapp.com.br/

What's can be wrong?

**Comment by jdkruzr** [comment]
I have this exact same problem. My instance is behind my main nginx reverse proxy, which terminates TLS and then passes connections to the Mastodon nginx reverse proxy behind it.

**Comment by AnibalDuarte** [comment]
I've just seen this configs on the top of the **/etc/nginx/sites-enabled/mastodon** file:

![image](https://user-images.githubusercontent.com/3143872/203579392-b68d355e-d62b-4aec-8303-078b959e2c11.png)

Should I open these ports (or any others than 80 and 443) on the Oracle's "firewall"?

**Comment by AnibalDuarte** [comment]
Today I've invested some time on this issue today and proceed as this:

- `rm -rf /home/mastodon/.rbenv`
- `rm -rf /home/mastodon/live`
- Uninstalled PostgreSQL
- Did all steps on https://docs.joinmastodon.org/admin/install/ (again)

And it still not working.

As I could check all /public folders are there, as the files (favicons, JS and CSS) but the browser is returning MIME type and 404 errors:

![image](https://user-images.githubusercontent.com/3143872/204109904-4cf8b2c4-1bc0-4bb6-b6ee-c4225e6de607.png)

**Comment by AnibalDuarte** [comment]
Can anyone help me with this or shoud I only quit trying?

**Comment by realrandolph** [comment]
I have had the same issue, but intermittently. How I fixed it was by increasing the allowable threads and connections per thread in my nginx config. My thinking was that it was happening as a result of nginx being overloaded (DoS style). I'm not sure if that was the case, but it drastically reduced the frequency with which I was seeing this error. Opening in a private browsing window can help as well, if you have the problem cached.

**Comment by AnibalDuarte** [comment]
I believe the cache nor the DDoS'ing is the problem, 'cause the service didn't work since I've installed it even with no users/access and I've tryed it on private window/another browser.
